openapi.yaml

```
@@ -141,6 +179,25 @@ components:
       properties:
         asyncapi:
           $ref: '#/components/schemas/AsyncAPIDocument'
+    DiffDocument:
+      type: object
+      required:
+        - asyncapi1
+        - asyncapi2
+      properties:
+        asyncapi1:
+          $ref: '#/components/schemas/AsyncAPIDocument'
+        asyncapi2:
+          $ref: '#/components/schemas/AsyncAPIDocument'
+    DiffResult:
+      type: object
+      properties:
+        equal:
+          type: boolean
+          description: Whether the two AsyncAPI documents are equal.
+        diff:
+          type: string
+          description: The diff between the two AsyncAPI documents.
```

Diff can be described by `yaml` format or JSON, so I opt to change the type to:

```suggestion
        diff:
          type: [object, string]
          description: The diff between the two AsyncAPI documents.
```